osc/fetch.py

```diff
@@ -221,12 +221,22 @@ def run(self, buildinfo):
                 cached += 1
                 if not i.name.startswith('container:') and i.pacsuffix != 'rpm':
                     continue
+
+                hdrmd5_is_valid = True
                 if i.hdrmd5:
                     if i.name.startswith('container:'):
                         hdrmd5 = dgst(i.fullfilename)
+                        if not hdrmd5 or hdrmd5 != i.hdrmd5:
+                            hdrmd5_is_valid = False
```

Comment:
pitnick: could be simplified to

if hdrmd5 != i.hdrmd5:

because None is also not equal, no?

Reply:
Changed.